### ISSUE TITLE
Unable to edit block config when updating layout of Service content type on its.uiowa.edu

### ISSUE THREAD

**Issue by e-marie-w** [uiowa/uiowa#8246]
# Problem
I am unable to edit the block config for for the Services content type layout. I can add new blocks just fine, but I am unable to edit them either after saving.

# Steps to reproduce
Pull down ITS locally and go to https://its.uiowa.ddev.site/admin/structure/types/manage/service/display/default/layout. See if you can edit the blocks. 

# Proposed solution
???

**Comment by e-marie-w** [comment]
I pulled down sandbox locally and started adding listblocks to the layout of a content type. After adding the block I can still edit its config, but when I try to save the layout I get 
`InvalidArgumentException: The configuration property third_party_settings.layout_builder.sections.2.components.10.configuration.sort.sticky.order doesn't exist. in Drupal\Core\Config\Schema\ArrayElement->get() (line 95 of core/lib/Drupal/Core/Config/Schema/ArrayElement.php).`
I'm not sure if this is part of the same problem or something else entirely. 

I tried different listblocks on different content types, it happens consistently. (The number of sections and components in the above error change to match the layout, but the rest of the error is the same).
